inductiva/molecules/scenarios/water_box/water_box.py

```diff
@@ -0,0 +1,145 @@
+        temperature: float = 300,
+        box_size: float = 3.0,
+    ):
+        """The scenario involves simulating a box filled with water molecules.
+        The simulation consists of three main steps: energy minimization, water 
+        molecule position decorrelation, and molecular dynamics simulation. 
+        Currently, this scenario exclusively supports the GROMACS simulator.
+        By default, for a given box size the water molecules are always 
+        initially positioned in the same locations following a uniform
+        distribution. 
+        To introduce randomness in the initial positions of the water molecules, 
+        a decorrelation step is performed. This step involves running a short 
+        simulation so that their position changes according to a normal 
+        distribution.
+        Args:
+            temperature: The temperature of the simulation in Kelvin.
+            box_size: The size of the box in nm.
+        """
+        self.template_dir = os.path.join(SCENARIO_TEMPLATE_DIR,
+                                         GROMACS_TEMPLATE_INPUT_DIR)
+        self.temperature = temperature
+        self.box_size = box_size
+
+    def simulate(
+            self,
+            simulator: Simulator = GROMACS(),
+            output_dir: Optional[Path] = None,
+            simulation_time: float = 10,  # ns
+            integrator: Literal["md", "sd", "bd"] = "md",
+            nsteps_minim: int = 5000):
+        """Simulate the water box scenario.
+
+        Args:
+            output_dir: The output directory to save the simulation results.
+            simulation_time: The simulation time in ns.
+            integrator: The integrator to use for the simulation. Options:
+                - "md" (Molecular Dynamics): Accurate leap-frog algorithm for 
+                integrating Newton's equations of motion.
+                - "sd" (Steepest Descent): Stochastic dynamics integrator with
+                leap-frog scheme.
+                - "bd" (Brownian Dynamics): Euler integrator for Brownian or 
+                position Langevin dynamics. 
+                
+            For more details on the integrators, refer to the GROMACS 
+            documentation at 
+            https://manual.gromacs.org/current/user-guide/mdp-options.html.
+
+            nsteps_minim: Number of steps for energy minimization.
+        """
+        self.nsteps = int(
+            simulation_time * 1e6 / 2
+        )  # convert to fs and divide by the time step of the simulation (2 fs)
+        self.integrator = integrator
+        self.nsteps_minim = nsteps_minim
+        replace_params_in_template(
+            self.template_dir, "commands.json.jinja",
+            {"box_size": self.box_size},
+            os.path.join(self.template_dir, "commands.json"))
+        commands = self.read_commands_from_file()
+        return super().simulate(simulator, output_dir, commands=commands)
+
+    def read_commands_from_file(self):
```

Comment:
Now that we see this will be used over most scenarios, we can set it in the scenario class, wdyt?

Reply:
the read_commands_from_file right? yes, I can move it :)